_posts/2020-07-30-intro-to-functional-programming-in-unity.markdown

```diff
@@ -0,0 +1,190 @@
+---
+layout: post
+title:  "Functional Programming in Unity: A Primer"
+date:   2020-07-30 00:00:00
+img: Unity_Technologies.png
+description: Unity Functional Programming Introduction
+author: Clock
+published: true
+---
+
+## Intro
+
+This is a unity (C#) friendly introduction to functional programming.
+
+This article assumes you know the bare basics of c# programming and are familiar with unity.
+
+By the end of this article you should learn the following:
+- A no bullshit description of functional programming
```

Comment:
So, this isn't something you knew before you wrote this, and it's not really something we thought about before this point either - but I talked about it with the guys and we figured that in the case we wanted to point professionals to the blog, we want to keep the blog posts similarly professional. Would you mind changing this line for me?

Good post, thanks Connor!

Reply:
I understand your concern and I have no problem changing this line to make the post more professional. Loop me into future conversations about content guidelines so I don't make the same mistake again.

Reply:
Absolutely, sorry we didn't think about it before this point. I'll try modifying the readme with some simple guidelines so folks can know beforehand.

Reply:
You're all merged in now!